defaults/main.yml

```
@@ -48,6 +48,7 @@ rsd_max_requests_github: "6"
 rsd_max_requests_gitlab: "6"
 rsd_max_requests_doi: "6"
 rsd_postgres_password: "changeme"
+rsd_postgres_authenticator_password: "ChangeMe"
```

I wonder if we should put it this way to not introduce breaking behavior? If people are not careful enough looking at a minor release, they might end up with a variable set to an insecure default.

```suggestion
rsd_postgres_authenticator_password: "{{ rsd_postgres_password }}"
```

We could keep the README as is to motivate setting the variable.

---

Thanks, you're of course right in this regard but thanks for changing!